### DISCUSSION TITLE
ZiGate Ethernet (wired) Gateway / Bridge open source HW based on ESP32

### DISCUSSION THREAD

**Discussion by Hedda** [zigpy/zigpy#790] [General]
ZiGate-Ethernet has been released as open source hardware by Frédéric Dubois a.k.a. fairecasoimeme and is available at zigate.fr

![image](https://user-images.githubusercontent.com/6320001/129525575-05dbac49-6782-4769-b31c-61e3a7113671.png)

https://github.com/fairecasoimeme/ZiGate-Ethernet

https://zigate.fr/produit/zigate-ethernet/

https://zigate.fr/documentation/descriptif-de-la-zigate-ethernet/

https://zigate.fr/documentation/installation-de-tasmota-pour-la-zigate-ethernet/

Note! This board ships without a Zigbee PCB radio board/antenna as standard so would at least need to add PiZiGate + option to use it:

https://zigate.fr/produit/pizigatev2/

![image](https://user-images.githubusercontent.com/6320001/129526590-036298d5-b689-417c-9159-bd2ed99b7202.png)

![image](https://user-images.githubusercontent.com/6320001/129687507-aa255542-d287-41cc-95b8-e1b87cd175a5.png)

ZiGate radio and this should work out-of-the-box with Home Assistant's ZHA integration via the zigpy-zigate radio library for zigpy

https://github.com/zigpy/zigpy-zigate

According to fairecasoimeme it should also be compatible with ESPHome using esphome-zbbridge as alternative firmware:

https://github.com/thegroove/esphome-zbbridge

https://github.com/fairecasoimeme/ZiGate/issues/366#issuecomment-898576244

Board is apparently sized to fit the enclosure case made for Raspberry Pi 4

![image](https://user-images.githubusercontent.com/6320001/129525814-bff49b63-1b7a-49ff-852d-9a667875a257.png)

**Comment by MattWestb** [comment]
Nice made !!

I wonder is its also working putting one RaspBee I / II on the header but i think ZHA have not implanted network connection for it in the radio lib (I think).

But one Elelabs PI HAT shall working out of the box if setting up Tasmota or ESPHome on the ESP32 OK :-)))

**Reply by Hedda** [reply]
> I wonder is its also working putting one RaspBee I / II on the header but i think ZHA have not implanted network connection for it in the radio lib (I think).

fairecasoimeme mentioned compatibility with Raspbee1/2 "and other RPI modules" but not sure if tested with those radio libs

https://github.com/fairecasoimeme/ZiGate/issues/366#issuecomment-898576244

> **_ZiGate-Ethernet is compatible with PiZiGate, Raspbee1/2 and other RPI modules with ZHA. (I'll do the doc)_**

Since the ESP32 essentially only provide a Serial-to-IP proxy functionality this should in theory also work with other Zigbee HATs/shields for Raspberry Pi, like RaspBee HAT by Dresden Elektronik, and ZigStar shield, or others too, such as for example:

https://zigate.fr/produit/pizigatev2/

https://phoscon.de/en/raspbee

https://elelabs.com/products/elelabs-zigbee-shield.html

https://www.popp.eu/zb-shield/

https://zig-star.com/projects/zigbee-shield/

https://www.tindie.com/products/GiovanniCas/zigbee-hat-with-cc2538-for-raspberry/

https://github.com/egony/cc2652p_E72-2G4M20S1E/wiki/Home-EN

https://github.com/egony/cc2652p_cc1352p_RF-STAR/wiki/Home-EN
